tests/parser/tst_parser.cpp

```diff
@@ -757,32 +757,32 @@ void tst_Parser::mapsAndArrays()
                    "{_ 1: [_ " + expected + "], \"Hello\": {_ " + expected + ": (_ )}}");
 }
 
-struct Input {
-    QByteArray data;
-    int consumed;
-};
-
 static const CborParserOperations byteArrayOps = {
     /* can_read_bytes = */ [](const CborValue *value, size_t len) {
-        auto input = static_cast<Input *>(value->parser->data.ctx);
-        return input->data.size() - input->consumed >= int(len);
+        auto data = static_cast<QByteArray *>(value->parser->data.ctx);
+        auto consumed = uintptr_t(value->source.token);
```

Comment:
I'd prefer the `uintptr_t` on the left side, but won't insist on your changing again.

Reply:
Yep, I've replaced the `auto` references.